packages/backend/package.json

```diff
@@ -40,6 +40,8 @@
     "@backstage/plugin-badges-backend": "workspace:^",
     "@backstage/plugin-catalog-backend": "workspace:^",
     "@backstage/plugin-code-coverage-backend": "workspace:^",
+    "@backstage/plugin-explore-backend": "workspace:^",
+    "@backstage/plugin-explore-common": "workspace:^",
```

Comment:
Oh, that must have been pulled over from the first implementation where I had the `ToolProvider` immplementation in the `plugins/explore.ts`. I could move the `getExampleTools` code into the `backend` as well, but it might bloat the `explore.ts` file a fair bit.